output/figure90.svg

```diff
@@ -0,0 +1,280 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
+  "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
Why is this committed? Isn't this going into one of the main figures?

Reply:
yes, I added it into figure8. If I run the model again the shape of it will probably be a little differentnt. If we are okay with not having the original figure here, I will remove it.